pfl/model/pytorch.py

```diff
@@ -160,22 +160,6 @@ def pytorch_model(self) -> torch.nn.Module:
     def variable_map(self) -> Dict[str, torch.Tensor]:
         return self._variable_map
 
-    @property
-    def central_optimizer_variable_map(
```

Comment:
This is removing a public interface, it should remain. If you still have this you shouldn't need to make changes in tests no?